lib/supavisor_web/controllers/tenant_controller.ex

```diff
@@ -99,6 +101,12 @@ defmodule SupavisorWeb.TenantController do
             tenant = Repo.preload(tenant, :users)
 
             with {:ok, %TenantModel{} = tenant} <- Tenants.update_tenant(tenant, tenant_params) do
+              for user <- tenant.users do
```

Comment:
Hmm, good call. We are not currently limiting the user count, but I believe we should, just in case, to avoid any unexpected behaviors. @chasers WDYT?

Reply:
this will be implemented as a separate task https://github.com/supabase/supavisor/issues/116